packages/oui-datagrid/src/cell/cell.controller.js

```diff
@@ -12,7 +12,13 @@ export default class {
 
         this.$element.css("display", "block");
 
-        this._compileCell();
+        if (this.row && this.row.$promise) {
```

Comment:
Suggestion:

```js
this.$q
	.when(null)
	.then(() => _(this.row).get("$promise"))
	.finally(() => this._compileCell());
```

Reply:
i like your suggestion but i think an "if" is more straightforward and easier to read in that case